### PR TITLE
BIGTOP-3735: Fix sqoop cannot found StringUtils from common-lang

### PR DESCRIPTION
**Pull request by kevinw66**
### For code changes:

- [ ] Does the title or this PR starts with the corresponding JIRA issue id (e.g. 'BIGTOP-3638. Your PR title ...')?
- [ ] Make sure that newly added files do not have any licensing issues. When in doubt refer to https://www.apache.org/licenses/

### CODE REVIEW

File: bigtop-packages/src/common/sqoop/patch2-SQOOP-3474.diff [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+       conf="common->default;redist->default"/>
+     <dependency org="org.kitesdk" name="kite-data-mapreduce" rev="${kite-data.version}"
+@@ -121,8 +123,6 @@ under the License.
+
+     <dependency org="com.google.code.p.arat" name="rat-lib"
+           rev="${rats-lib.version}" conf="releaseaudit->default" />
+-    <dependency org="commons-lang" name="commons-lang"
+-      rev="${commons-lang.version}" conf="releaseaudit->default"/>
+     <dependency org="commons-collections" name="commons-collections"
+       rev="${commons-collections.version}" conf="releaseaudit->default"/>
```
Comment (iwasakims) [inline bigtop-packages/src/common/sqoop/patch2-SQOOP-3474.diff:21]: Could you rename the patch to the name like `patch2-redist-commons-lang.diff`?  We usually mark the patch with JIRA id if it is the one submitted/committed/backported. There is no patch submitted to SQOOP-3474 and it was originally reported as issue of sqoop2-client.

**Comment by iwasakims** [comment]
@kevinw66 Could you increment the release number to 2 in bigtop.bom?  The package name will be updated to `sqoop-1.4.7-2.el7.noarch.rpm`.

```
    'sqoop' {
      name    = 'sqoop'
      relNotes = 'Apache Sqoop v1'
      version { base = '1.4.7'; pkg = base; release = 1 }
```

**Comment by iwasakims** [comment]
The fix worked for me on manual testing. I guess commons-lang disappeared from transitive dependencies.

```
[root@cf5370850071 /]# sqoop import --connect jdbc:mysql://localhost/test --table test
...
2022-07-13 16:45:29,826 INFO mapreduce.ImportJobBase: Transferred 12 bytes in 21.0065 seconds (0.5713 bytes/sec)
2022-07-13 16:45:29,829 INFO mapreduce.ImportJobBase: Retrieved 2 records.
```

**Comment by kevinw66** [comment]
> @kevinw66 Could you increment the release number to 2 in bigtop.bom? The package name will be updated to `sqoop-1.4.7-2.el7.noarch.rpm`.
> 
> ```
>     'sqoop' {
>       name    = 'sqoop'
>       relNotes = 'Apache Sqoop v1'
>       version { base = '1.4.7'; pkg = base; release = 1 }
> ```

Updated, thanks @iwasakims

**Review by iwasakims — APPROVED** [review]
+1. Thanks for the contribution, @kevinw66.

**Comment by iwasakims** [comment]
I merged this to branch-3.1 and **master**. While we can not test it until [BIGTOP-3737](https://issues.apache.org/jira/browse/BIGTOP-3737) is resolved, I'm assuming master is affected too.

**Pull request merged**